egui_demo_lib/src/apps/color_test.rs

```diff
@@ -36,14 +36,6 @@ impl epi::App for ColorTest {
 
     fn update(&mut self, ctx: &egui::CtxRef, frame: &mut epi::Frame<'_>) {
         egui::CentralPanel::default().show(ctx, |ui| {
-            if frame.is_web() {
-                ui.colored_label(
-                    RED,
-                    "NOTE: The WebGL backend does NOT pass the color test."
-                );
-                ui.small("This is because WebGL does not support a linear framebuffer blending (not even WebGL2!).\nMaybe when WebGL3 becomes mainstream in 2030 the web can finally get colors right?");
-                ui.separator();
-            }
             ScrollArea::auto_sized().show(ui, |ui| {
```

Comment:
Ideally we should still show a warning about WebGL1 not passing the color test. Right now there is no way to query that, but perhaps a milder warning would fit here:

``` rust
if frame.is_web() {
    ui.label("NOTE: the WebGL1 backend does not pass the color test, but the WebGL2 backend does.");
}
```